pkg/aws/api_interface/cloudformation_api_client.go

```diff
@@ -0,0 +1,91 @@
+package aws
```

Comment:
@den-rgb As a way of making the review smaller, everything in this package and the tests could be introduced as a separate PR. We'd just drop in the new implementations but not update anything to use them yet. Just a thought.